handbook/manifesto.md

```diff
@@ -47,21 +47,33 @@ As a club, Launch Pad supports every member’s experience through multiple ways
 
 ### Club Structure
 
+#### Co-Presidents
+
+Launch Pad is lead by two co-presidents. The role of the co-presidents is to act as the face of the club, manage our resources and accounts, and help the leadership team feel empowered and supported in initiatives they want to drive. The co-presidents do this by providing the leadership team with whatever guidance and resources you need, such as contacts, accounts, money, advice, and so on. Co-presidents also help ensure Launch Pad's initiatives and activities are aligned with the club's mission and objectives.
```

Comment:
```suggestion
Launch Pad is lead by two co-presidents. The role of the co-presidents is to act as the face of the club, manage our resources and accounts, and help the leadership team feel empowered and supported in initiatives they want to drive. The co-presidents do this by providing the leadership team with whatever guidance and resources they need, such as contacts, accounts, money, advice, and so on. Co-presidents also help ensure Launch Pad's initiatives and activities are aligned with the club's mission and objectives.
```

Reply:
@tarikeshaq is there a change in this comment? :o

Reply:
😂 Sorry the block doesn't show it for some reason, but the "you need" -> "they need"

Reply:
ah i see it now, thanks!